compose/services/docker-compose.database.yml

```
@@ -10,11 +10,18 @@ services:
+      MARIADB_DATABASE: ${DB_DATABASE}
+      MARIADB_USER: ${DB_USERNAME}
+      MARIADB_PASSWORD: ${DB_PASSWORD}
     networks:
-     - internal
-     - web
\ No newline at end of file
+      - internal
+      - web
+    
+    healthcheck:
+      test: [ "CMD", "mysqladmin", "ping", "-h", "localhost", "-u",  "$$DB_USERNAME", "-p$$DB_PASSWORD" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)